### PR TITLE
KAS-3386 filter on all other statuses

### PR DESCRIPTION
**Pull request by ValenberghsSven**
To filter out 1 status, I included all the other status id's in the filter.

### CODE REVIEW

File: app/pods/publications/overview/urgent/route.js [hunk 1]
```diff
@@ -9,20 +22,28 @@ export default class PublicationsOverviewUrgentRoute extends PublicationsOvervie
...
   ];
-  tableConfigStorageKey = "publication-table.urgent";
+  tableConfigStorageKey = 'publication-table.urgent';
 
   beforeModel() {
-    super.beforeModel(...arguments)
+    super.beforeModel(...arguments);
+    const notPublishedStatuses = this.store
+      .peekAll('publication-status')
+      .filter((status) => {
```
Comment (erikap) [inline app/pods/publications/overview/urgent/route.js]: It’s maybe easier to understand the intention of the filter by just [rejecting](https://api.emberjs.com/ember/3.28/classes/EmberArray/methods/reject?anchor=rejectBy) the status where `status.isPublished` is true?

Comment (ValenberghsSven) [reply app/pods/publications/overview/urgent/route.js]: Never used rejectBy before, didn't know it existed.
I probably could have reverse filtered with a normal filter tbh, I copy pasted the filtering from different route.

This looks way better though.
Thanks!

**Review by erikap — APPROVED** [review]

**Comment by erikap** [comment]
Not sure whether the failing tests are related to the change in this PR. I don't expect so, but a retrigger of the Jenkins build made even more tests fail :see_no_evil:  Can you have a look @ValenberghsSven ?  Beside the tests, this PR looks good :+1:

**Comment by erikap** [comment]
Failed tests are not related to this PR.

**Pull request merged**